### ISSUE TITLE
Using addon in app causes missing module error caused by ember-auto-import

### ISSUE THREAD

**Issue by oracle1wr** [adopted-ember-addons/ember-impagination#261]
ember-cli: 3.8.1
node: 8.14.0

```
Could not find module `impagination` imported from `***/components/impagination-dataset
```

**Comment by Alonski** [comment]
Thanks for the report.
This is an issue with the way we import and then export `impagination`
`Ember Auto Import` doesn't seem to register that `impagination` needs to be bundled.
There is an open issue: https://github.com/ef4/ember-auto-import/issues/209

There is a workaround that you can do for now:
1. `yarn add ember-auto-import -D`
2. `yarn add impagination -D`

This will install the dependencies into your project and will cause `ember-auto-import` to bundle `impagination` with your app.

**Comment by oracle1wr** [comment]
thanks, it worked
At least I don't see the error anymore.

**Comment by Alonski** [comment]
Reopened since this is still an issue 🙂

**Comment by Alonski** [comment]
@oracle1wr Could you try pointing to this branch to see if it fixes the issue: https://github.com/adopted-ember-addons/ember-impagination/pull/263

**Comment by oracle1wr** [comment]
Changes in #263 fixed the issue for me. Now it works as expected.

**Comment by Alonski** [comment]
Awesome. I will merge and publish a release over the weekend. Thanks!

**Comment by zion** [comment]
Is there an ETA on this getting merged? I know addon maintainers are busy, just curious... 

Id like to try out this addon, but I'd rather not add another addon just for the workaround.

**Comment by Alonski** [comment]
@zion Thanks for the reminder!
Just released 1.0.0
https://github.com/adopted-ember-addons/ember-impagination/releases/tag/v1.0.0
